### ISSUE TITLE
scene.skyboxMip affecting skybox without prefiltered data.

### ISSUE THREAD

**Issue by Maksims** [playcanvas/engine#294]
`skyboxMip` defines prefiltered value to choose from for skybox.
First of all it would be good to make it not int but floating number, so value can be changed gradually, just like glossiness on material.

Second it has bug when scene has non-prefiltered skybox, it is affecting skybox in accordance to seams workaround, just play with mip level on skybox without prefiltered data to get clear idea of a problem.
Basically it should be ignored if skybox don't have prefiltered data.

And please add documentation for `skyboxIntensity` and `skyboxMip` so it appears on developer reference website.

**Comment by guycalledfrank** [comment]
Skybox uses tex.fixCubemapSeams flag as materials do. You probably set it to true when it's not supposed to be true.

**Comment by guycalledfrank** [comment]
Also, skyboxMip is not supposed to work without prefiltering, as auto-generated mips look terrible anyway.
